external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -175,7 +175,7 @@ pub enum DefInfo {
         /// Type args
         Vec<Type>,
         /// Arg names
-        Vec<Symbol>,
+        Vec<Name>,
```

Comment:
Same question here (and below) about the change from symbols to names: are we using this location information for a specific purpose?

Reply:
Ah, I see, here (at least) we need them for the inlay hints. I suppose the other API changes are just to make things more-uniform?

Reply:
Indeed! @rvantonder suggested adding on-hover to param inlay hints and I needed locations of parameters in the function's signature (and also indeed, the fields location was added to avoid doing the conversions)